lib/widgets/molecules/resources_section.dart

```diff
@@ -1,54 +1,82 @@
+// ignore_for_file: non_constant_identifier_names
+
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:mm_flutter_app/constants/app_constants.dart';
-import 'package:mm_flutter_app/widgets/atoms/section_tile.dart';
 
-import 'image_tile.dart';
+class ResourceImage extends StatelessWidget {
+  final AssetImage resource_image;
+  final String subheading;
+
+  const ResourceImage({
+    Key? key,
+    required this.resource_image,
+    required this.subheading,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    return Padding(
+      padding: const EdgeInsets.only(
+          top: Insets.paddingSmall, bottom: Insets.paddingSmall),
+      child: Column(
+        children: [
+          Image(image: resource_image),
+          Padding(
+            padding: const EdgeInsets.only(top: Insets.paddingSmall),
+            child: Text(subheading,
+                style: theme.textTheme.titleSmall?.copyWith(
+                  color: Colors.black,
```

Comment:
Please don't use hardcoded colors. I know that this is what UX used, but we should know how to translate these to their equivalent color scheme value. **onSurface** is probably the best color role to use here.